shell.nix

```diff
@@ -1,28 +1,41 @@
-{ pkgs ? import (builtins.fetchTarball {
-    url = "https://github.com/NixOS/nixpkgs/archive/e7603eba51f2c7820c0a182c6bbb351181caa8e7.zip";
-    sha256 = "sha256:0mwck8jyr74wh1b7g6nac1mxy6a0rkppz8n12andsffybsipz5jw";
-  }) { } }:
+{
+  pkgs ? import ./nix/pkgs.nix
+}:
 
 let
-  qtCustom = with pkgs.qt515;
+
+  qtCustom = (with pkgs.qt515;
     env "qt-custom-${qtbase.version}" ([
+## TODO:check again after Qt upgrade
+      qtbase
+      qtdeclarative
+      qtquickcontrols
       qtquickcontrols2
+      qtsvg
+      qtmultimedia
+      qtwebview
+      qttools
+      qtwebchannel
       qtgraphicaleffects
-      qtbase qttools qtdeclarative
-      qtlottie qtmultimedia
-      qtquickcontrols qtquickcontrols2
-      qtsvg qtwebengine qtwebview
-    ]);
+      qtwebengine
+      qtlocation
+#      qtlottie # return after qt upgrade ?
```

Comment:
What?

Reply:
`qtlottie` is missing in 5.15.2

Reply:
Actually I removed it, not required for the build at least.